### ISSUE TITLE
Crash report when putting two items together (modpack)

### ISSUE THREAD

**Issue by k1r0s** [breakinblocks/Plonk#1]
nice idea for the mod.

I have a crash report

steps to reproduce:

create empty world

diamond chestplate next to diamond sword, in the same block.

```
---- Minecraft Crash Report ----
// Don't be sad. I'll do better next time, I promise!

Time: 2020-10-16 21:28:48 CEST
Description: Unexpected error

java.lang.NullPointerException
    at net.minecraft.client.renderer.RenderGlobal.drawSelectionBox(RenderGlobal.java:2824)
    at net.minecraft.client.renderer.EntityRenderer.renderWorldPass(EntityRenderer.java:1822)
    at net.minecraft.client.renderer.EntityRenderer.renderWorld(EntityRenderer.java:1614)
    at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1385)
    at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1119)
    at net.minecraft.client.Minecraft.run(Minecraft.java:3942)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Affected level --
  Level name: MpServer
  All players: 1 total; [EntityPlayerSP['mori'/304, l='MpServer', x=182.50, y=77.00, z=257.25]]
  Chunk stats: MultiplayerChunkCache: 240, 240
  Level seed: 0
  Level generator: ID 00 - default, ver 1. Features enabled: false
  Level generator options: 
  Level spawn location: World: (196,64,256), Chunk: (at 4,4,0 in 12,16; contains blocks 192,0,256 to 207,255,271), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
  Level time: 1048 game time, 1048 day time
  Level dimension: 0
  Level storage version: 0x00000 - Unknown?
  Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
  Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
  Forced entities: 112 total; [EntitySkeleton['Skeleton'/521, l='MpServer', x=165.52, y=38.00, z=252.70], EntitySpider['Spider'/525, l='MpServer', x=168.50, y=38.00, z=247.50], EntityZombie['Zombie'/526, l='MpServer', x=186.50, y=24.00, z=282.50], EntitySkeleton['Skeleton'/532, l='MpServer', x=220.49, y=36.00, z=289.33], EntitySkeleton['Skeleton'/533, l='MpServer', x=235.50, y=56.00, z=257.50], EntityCreeper['Creeper'/537, l='MpServer', x=182.62, y=38.00, z=250.22], EntitySkeleton['Skeleton'/549, l='MpServer', x=188.50, y=38.00, z=238.50], EntitySkeleton['Skeleton'/550, l='MpServer', x=191.50, y=38.00, z=237.50], EntityZombie['Zombie'/551, l='MpServer', x=216.50, y=34.00, z=286.50], EntityCreeper['Creeper'/557, l='MpServer', x=197.50, y=39.00, z=207.50], EntitySkeleton['Skeleton'/558, l='MpServer', x=170.66, y=38.00, z=247.96], EntitySpider['Velma'/306, l='MpServer', x=171.02, y=38.81, z=245.96], EntityPlayerSP['mori'/304, l='MpServer', x=182.50, y=77.00, z=257.25], EntityItem['item.item.quark:root_blue_flower'/309, l='MpServer', x=183.10, y=45.00, z=271.88], EntityZombie['Zombie'/565, l='MpServer', x=176.50, y=40.00, z=245.50], EntityOwl['Owl'/310, l='MpServer', x=185.50, y=67.00, z=224.50], EntityOwl['Owl'/311, l='MpServer', x=187.50, y=66.00, z=222.50], EntityZombie['Zombie'/567, l='MpServer', x=164.77, y=38.00, z=252.05], EntityZombie['Zombie'/568, l='MpServer', x=184.50, y=38.00, z=252.50], EntityZombie['Zombie'/569, l='MpServer', x=199.80, y=33.00, z=226.48], EntityBat['Bat'/314, l='MpServer', x=191.25, y=11.10, z=277.75], EntityCreeper['Creeper'/570, l='MpServer', x=179.78, y=27.00, z=275.54], EntityBat['Bat'/315, l='MpServer', x=162.59, y=40.12, z=270.78], EntitySpider['Spider'/571, l='MpServer', x=208.29, y=32.00, z=286.72], EntityBat['Bat'/316, l='MpServer', x=193.46, y=30.94, z=286.48], EntityBat['Bat'/317, l='MpServer', x=192.60, y=26.47, z=274.58], EntityZombie['Zombie'/575, l='MpServer', x=193.16, y=42.00, z=253.48], EntityCreeper['Creeper'/577, l='MpServer', x=151.50, y=16.00, z=279.50], EntityCow['Cow'/323, l='MpServer', x=231.50, y=66.00, z=283.50], EntityCow['Cow'/324, l='MpServer', x=229.50, y=66.00, z=284.50], EntityCreeper['Creeper'/580, l='MpServer', x=145.50, y=42.00, z=246.50], EntityCow['Cow'/325, l='MpServer', x=228.50, y=66.00, z=284.50], EntityCow['Cow'/326, l='MpServer', x=229.50, y=66.00, z=282.50], EntityBat['Bat'/327, l='MpServer', x=182.60, y=12.01, z=314.61], EntityWolf['Wolf'/328, l='MpServer', x=175.58, y=68.00, z=215.50], EntityWolf['Wolf'/329, l='MpServer', x=186.50, y=66.00, z=215.50], EntityCreeper['Creeper'/585, l='MpServer', x=145.50, y=42.00, z=243.50], EntityWolf['Wolf'/330, l='MpServer', x=184.50, y=66.00, z=215.50], EntityWolf['Wolf'/331, l='MpServer', x=178.72, y=66.00, z=208.45], EntityCreeper['Creeper'/587, l='MpServer', x=191.50, y=40.00, z=229.50], EntityWolf['Wolf'/332, l='MpServer', x=196.50, y=64.00, z=205.50], EntityCreeper['Creeper'/588, l='MpServer', x=193.49, y=40.00, z=223.83], EntityWolf['Wolf'/333, l='MpServer', x=198.50, y=64.00, z=206.50], EntityWolf['Wolf'/334, l='MpServer', x=193.50, y=64.00, z=206.50], EntityCreeper['Creeper'/590, l='MpServer', x=178.50, y=45.00, z=266.50], EntityWolf['Wolf'/335, l='MpServer', x=197.50, y=64.00, z=205.50], EntityArchaeologist['Archaeologist'/336, l='MpServer', x=181.00, y=37.00, z=299.00], EntityItem['item.item.quark:root_blue_flower'/339, l='MpServer', x=176.88, y=16.00, z=294.45], EntityCaveSpider['Greg the Mighty'/340, l='MpServer', x=136.50, y=22.00, z=267.50], EntityOwl['Owl'/342, l='MpServer', x=241.50, y=64.00, z=292.50], EntityOwl['Owl'/343, l='MpServer', x=241.50, y=64.00, z=293.50], EntityOwl['Owl'/344, l='MpServer', x=241.50, y=64.00, z=294.50], EntitySpider['Spider'/600, l='MpServer', x=211.71, y=32.00, z=285.37], EntityOwl['Owl'/345, l='MpServer', x=240.50, y=64.00, z=293.50], EntitySpider['Spider'/601, l='MpServer', x=215.41, y=34.00, z=288.07], EntitySpider['Spider'/602, l='MpServer', x=213.50, y=32.00, z=284.50], EntityZombie['Zombie'/603, l='MpServer', x=189.50, y=38.00, z=219.50], EntityCreeper['Creeper'/604, l='MpServer', x=190.50, y=38.00, z=221.50], EntitySkeleton['Skeleton'/612, l='MpServer', x=173.28, y=36.00, z=260.59], EntityCreeper['Creeper'/613, l='MpServer', x=185.57, y=38.00, z=229.18], EntityCreeper['Creeper'/615, l='MpServer', x=157.84, y=32.50, z=296.47], EntityItem['item.item.stick'/360, l='MpServer', x=169.80, y=36.00, z=297.57], EntitySnake['Snake'/361, l='MpServer', x=152.50, y=70.00, z=299.50], EntitySkeleton['Skeleton'/363, l='MpServer', x=196.50, y=21.00, z=292.50], EntitySpider['Spider'/364, l='MpServer', x=193.50, y=38.00, z=214.50], EntityZombie['Zombie'/366, l='MpServer', x=158.48, y=38.00, z=259.25], EntityCreeper['Creeper'/367, l='MpServer', x=180.50, y=38.00, z=250.50], EntityCow['Cow'/369, l='MpServer', x=134.50, y=70.00, z=259.50], EntityCow['Cow'/370, l='MpServer', x=135.50, y=71.00, z=261.50], EntityCow['Cow'/371, l='MpServer', x=135.50, y=71.00, z=259.50], EntityCow['Cow'/372, l='MpServer', x=135.50, y=71.00, z=260.50], EntityItem['item.tile.doublePlant.syringa'/373, l='MpServer', x=120.49, y=64.00, z=247.09], EntityZombie['Maximillian the Mean'/375, l='MpServer', x=135.45, y=30.00, z=298.61], EntityBat['Bat'/383, l='MpServer', x=174.37, y=29.17, z=296.33], EntityZombie['Zombie'/384, l='MpServer', x=148.50, y=45.00, z=263.50], EntityZombie['Zombie'/385, l='MpServer', x=142.75, y=45.00, z=262.52], EntityItem['item.item.quark:root'/387, l='MpServer', x=173.91, y=46.00, z=314.95], EntitySkeleton['Mister Rob'/389, l='MpServer', x=211.66, y=32.00, z=304.74], EntitySkeleton['Skeleton'/391, l='MpServer', x=143.69, y=33.00, z=263.53], EntityZombie['Zombie'/392, l='MpServer', x=167.20, y=38.00, z=249.54], EntityItem['item.item.quark:root_blue_flower'/393, l='MpServer', x=155.15, y=33.00, z=199.96], EntityBat['Bat'/650, l='MpServer', x=184.28, y=44.12, z=336.28], EntityCreeper['Creeper'/395, l='MpServer', x=198.49, y=40.00, z=208.83], EntityTroll['Troll'/398, l='MpServer', x=216.50, y=32.00, z=281.50], EntityZombieVillager['Zombie Villager'/655, l='MpServer', x=167.50, y=31.00, z=291.50], EntityZombieVillager['Zombie Villager'/656, l='MpServer', x=169.55, y=31.50, z=291.84], EntityZombie['Shane the Cracked'/400, l='MpServer', x=262.59, y=21.00, z=236.52], EntityCreeper['Yu'/402, l='MpServer', x=106.46, y=23.08, z=249.74], EntitySpectre['Spectre'/665, l='MpServer', x=113.50, y=22.00, z=320.50], EntityZombie['Zombie'/667, l='MpServer', x=221.50, y=39.00, z=185.50], EntityCreeper['Creeper'/413, l='MpServer', x=171.77, y=72.00, z=252.47], EntityCreeper['Creeper'/677, l='MpServer', x=241.50, y=45.00, z=329.50], EntitySkeleton['Skeleton'/684, l='MpServer', x=244.50, y=46.00, z=331.54], EntitySkeleton['Skeleton'/687, l='MpServer', x=171.50, y=36.00, z=264.50], EntityCreeper['Creeper'/434, l='MpServer', x=196.50, y=21.00, z=289.50], EntityBat['Bat'/438, l='MpServer', x=199.75, y=32.70, z=293.55], EntityCow['Cow'/441, l='MpServer', x=251.68, y=63.00, z=194.31], EntityCow['Cow'/442, l='MpServer', x=251.82, y=63.00, z=190.52], EntityCow['Cow'/443, l='MpServer', x=247.73, y=63.00, z=200.17], EntityCreeper['Creeper'/444, l='MpServer', x=178.50, y=34.00, z=284.50], EntityCreeper['Creeper'/445, l='MpServer', x=177.47, y=34.00, z=284.00], EntityCreeper['Creeper'/446, l='MpServer', x=174.80, y=34.00, z=280.56], EntityBat['Bat'/458, l='MpServer', x=147.40, y=41.76, z=291.44], EntityTroll['Troll'/472, l='MpServer', x=177.50, y=32.00, z=274.50], EntityBat['Bat'/479, l='MpServer', x=173.27, y=40.10, z=252.19], EntityZombie['Zombie'/485, l='MpServer', x=178.50, y=35.00, z=262.50], EntityCreeper['Creeper'/487, l='MpServer', x=235.50, y=56.00, z=255.50], EntityBat['Bat'/499, l='MpServer', x=188.34, y=22.91, z=280.43], EntityTroll['Troll'/504, l='MpServer', x=190.50, y=38.00, z=218.50], EntitySpider['Spider'/505, l='MpServer', x=193.50, y=38.00, z=209.50], EntityZombie['Zombie'/507, l='MpServer', x=215.20, y=31.00, z=271.51], EntitySkeleton['Skeleton'/509, l='MpServer', x=175.50, y=41.00, z=305.50]]
  Retry entities: 0 total; []
  Server brand: fml,forge
  Server type: Integrated singleplayer server

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Windows 7 (amd64) version 6.1
  Java Version: 1.8.0_261, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 549191216 bytes (523 MB) / 2074607616 bytes (1978 MB) up to 2863661056 bytes (2731 MB)
  JVM Flags: 2 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx3G
  IntCache: cache: 1, tcache: 1, allocated: 12, tallocated: 94
  FML: MCP 9.42 Powered by Forge 14.23.5.2854 Optifine OptiFine_1.12.2_HD_U_F5 58 mods loaded, 58 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State  | ID                   | Version             | Source                                        | Signature                                |
       |:------ |:-------------------- |:------------------- |:--------------------------------------------- |:---------------------------------------- |
       | LCHIJA | minecraft            | 1.12.2              | minecraft.jar                                 | None                                     |
       | LCHIJA | mcp                  | 9.42                | minecraft.jar                                 | None                                     |
       | LCHIJA | FML                  | 8.0.99.99           | forge-1.12.2-14.23.5.2854.jar                 | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | forge                | 14.23.5.2854        | forge-1.12.2-14.23.5.2854.jar                 | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | otgcore              | 1.12.2 - v8.3       | minecraft.jar                                 | None                                     |
       | LCHIJA | mantle               | 1.12-1.3.3.55       | Mantle-1.12-1.3.3.55.jar                      | None                                     |
       | LCHIJA | inspirations         | 1.12.2-0.2.9        | Inspirations-1.12.2-0.2.9.jar                 | None                                     |
       | LCHIJA | placebo              | 1.6.0               | Placebo-1.12.2-1.6.0.jar                      | None                                     |
       | LCHIJA | apotheosis           | 1.12.4              | Apotheosis-1.12.2-1.12.5.jar                  | None                                     |
       | LCHIJA | jei                  | 4.15.0.291          | jei_1.12.2-4.15.0.291.jar                     | None                                     |
       | LCHIJA | appleskin            | 1.0.9               | AppleSkin-mc1.12-1.0.9.jar                    | None                                     |
       | LCHIJA | astikorcarts         | 1.12.2-0.1.2.7      | astikorcarts-1.12.2-0.1.2.7.jar               | None                                     |
       | LCHIJA | tombstone            | 4.0.0               | tombstone-4.0.0-1.12.2.jar                    | None                                     |
       | LCHIJA | quark                | r1.6-179            | Quark-r1.6-179.jar                            | None                                     |
       | LCHIJA | autoreglib           | 1.3-32              | AutoRegLib-1.3-32.jar                         | None                                     |
       | LCHIJA | baubles              | 1.5.2               | Baubles-1.12-1.5.2.jar                        | None                                     |
       | LCHIJA | forgelin             | 1.8.4               | Forgelin-1.8.4.jar                            | None                                     |
       | LCHIJA | betterfoliage        | 2.3.1               | BetterFoliage-MC1.12-2.3.1.jar                | None                                     |
       | LCHIJA | patchouli            | 1.0-21              | Patchouli-1.0-21.jar                          | None                                     |
       | LCHIJA | bewitchment          | 0.22.40             | bewitchment-1.12.2-0.0.22.40.jar              | None                                     |
       | LCHIJA | bibliocraft          | 2.4.5               | BiblioCraft[v2.4.5][MC1.12.2].jar             | None                                     |
       | LCHIJA | biomecolorizer       | 1.0.0               | BiomeColorizer-1.12.2-v1.0.1.jar              | None                                     |
       | LCHIJA | ceramics             | 1.12-1.3.7          | Ceramics-1.12-1.3.7.jar                       | None                                     |
       | LCHIJA | cg                   | 1.0                 | cg_1.12-1.0.jar                               | None                                     |
       | LCHIJA | chameleon            | 1.12-4.1.3          | Chameleon-1.12-4.1.3.jar                      | None                                     |
       | LCHIJA | charm                | 1.12.2-1.3.7        | Charm-1.12.2-1.3.7.jar                        | None                                     |
       | LCHIJA | chunkanimator        | 1.2                 | ChunkAnimator-MC1.12-1.2.jar                  | None                                     |
       | LCHIJA | clumps               | 3.1.2               | Clumps-3.1.2.jar                              | None                                     |
       | LCHIJA | comforts             | 1.4.1.3             | comforts-1.12.2-1.4.1.3.jar                   | 2484ef4d131fdc0dca0647aa21b7b944ddb935a1 |
       | LCHIJA | cookingforblockheads | 6.5.0               | CookingForBlockheads_1.12.2-6.5.0.jar         | None                                     |
       | LCHIJA | craftablesaddles     | 1.3                 | Craftable+Saddles[1.12]-1.3.jar               | None                                     |
       | LCHIJA | orelib               | 3.6.0.1             | OreLib-1.12.2-3.6.0.1.jar                     | 7a2128d395ad96ceb9d9030fbd41d035b435753a |
       | LCHIJA | dsurround            | 3.6.1.0             | DynamicSurroundings-1.12.2-3.6.1.0.jar        | 7a2128d395ad96ceb9d9030fbd41d035b435753a |
       | LCHIJA | waila                | 1.8.26              | Hwyla-1.8.26-B41_1.12.2.jar                   | None                                     |
       | LCHIJA | llibrary             | 1.7.19              | llibrary-1.7.19-1.12.2.jar                    | b9f30a813bee3b9dd5652c460310cfcd54f6b7ec |
       | LCHIJA | iceandfire           | 1.9.1               | iceandfire-1.9.1-1.12.2.jar                   | None                                     |
       | LCHIJA | ironbackpacks        | 1.12.2-3.0.8-12     | IronBackpacks-1.12.2-3.0.8-12.jar             | None                                     |
       | LCHIJA | journeymap           | 1.12.2-5.7.1        | journeymap-1.12.2-5.7.1.jar                   | None                                     |
       | LCHIJA | jeresources          | 0.9.2.60            | JustEnoughResources-1.12.2-0.9.2.60.jar       | None                                     |
       | LCHIJA | testdummy            | 1.12                | MmmMmmMmmMmm-1.12-1.14.jar                    | None                                     |
       | LCHIJA | mobends              | 1.0.0-beta-20.06.20 | MoBends_1.12.2-1.0.0-beta-20.06.20.jar        | None                                     |
       | LCHIJA | natura               | 1.12.2-4.3.2.69     | natura-1.12.2-4.3.2.69.jar                    | None                                     |
       | LCHIJA | netherite1_12_2      | 1.1                 | Netherite(1.12.2).jar                         | None                                     |
       | LCHIJA | openterraingenerator | v8.3                | OpenTerrainGenerator-1.12.2+-+v8.3.jar        | None                                     |
       | LCHIJA | plonk                | 1.12.2-1.0.0        | plonk-1.12.2-1.0.0.jar                        | None                                     |
       | LCHIJA | roguelike            | 1.8.0               | RoguelikeDungeons-1.12.2-1.8.0.jar            | None                                     |
       | LCHIJA | rustic               | 1.1.6               | rustic-1.1.6.jar                              | None                                     |
       | LCHIJA | soundphysics         | 1.0.8-1             | Sound-Physics-1.12.2.jar                      | None                                     |
       | LCHIJA | spartanweaponry      | beta 1.3.8          | SpartanWeaponry-1.12.2-beta-1.3.8.jar         | None                                     |
       | LCHIJA | spartanfire          | 0.08                | spartanfire-0.08.jar                          | None                                     |
       | LCHIJA | storagedrawers       | 5.2.2               | StorageDrawers-1.12.2-5.4.2.jar               | None                                     |
       | LCHIJA | travelersdream       | 1.1                 | Traveler'sDream-Compact-1.12.2-v1.2.jar       | None                                     |
       | LCHIJA | vanillafix           | 1.0.10-150          | VanillaFix-1.0.10-150.jar                     | None                                     |
       | LCHIJA | veinminer            | 0.38.2              | VeinMiner-1.12-0.38.2.647+b31535a.jar         | None                                     |
       | LCHIJA | veinminermodsupport  | 0.38.2              | VeinMiner-1.12-0.38.2.647+b31535a.jar         | None                                     |
       | LCHIJA | wolfarmor            | 3.4.1.1             | wolfarmor-1.12.2-3.4.1.1-UNIVERSAL-signed.jar | None                                     |
       | LCHIJA | worleycaves          | 1.5.1               | worleycaves-1.5.1.jar                         | None                                     |
       | LCHIJA | ironagefurniture     | 0.1.0               | iron-age-furniture-1.12.2-0.1.0.1.jar         | None                                     |
  Loaded coremods (and transformers): TransformLoader (DynamicSurroundings-1.12.2-3.6.1.0.jar)
                                        
                                      ForgelinPlugin (Forgelin-1.8.4.jar)
                                        
                                      OTGCorePlugin (OTG-Core.jar)
                                        com.pg85.otg.forge.asm.excluded.OTGClassTransformer
                                      Quark Plugin (Quark-r1.6-179.jar)
                                        vazkii.quark.base.asm.ClassTransformer
                                      BewitchmentFMLLoadingPlugin (bewitchment-1.12.2-0.0.22.40.jar)
                                        
                                      ApotheosisCore (Apotheosis-1.12.2-1.12.5.jar)
                                        shadows.ApotheosisTransformer
                                      iceandfire (iceandfire-1.9.1-1.12.2.jar)
                                        com.github.alexthe666.iceandfire.patcher.IceAndFireRuntimePatcher
                                      llibrary (llibrary-core-1.0.11-1.12.2.jar)
                                        net.ilexiconn.llibrary.server.core.plugin.LLibraryTransformer
                                        net.ilexiconn.llibrary.server.core.patcher.LLibraryRuntimePatcher
                                      CharmLoadingPlugin (Charm-1.12.2-1.3.7.jar)
                                        svenhjol.charm.base.CharmClassTransformer
                                      BetterFoliageLoader (BetterFoliage-MC1.12-2.3.1.jar)
                                        mods.betterfoliage.loader.BetterFoliageTransformer
                                      VanillaFixLoadingPlugin (VanillaFix-1.0.10-150.jar)
                                        
                                      WolfArmorCore (wolfarmor-1.12.2-3.4.1.1-UNIVERSAL-signed.jar)
                                        
                                      CoreModLoader (Sound-Physics-1.12.2.jar)
                                        com.sonicether.soundphysics.CoreModInjector
                                      LoadingPlugin (ChunkAnimator-MC1.12-1.2.jar)
                                        lumien.chunkanimator.asm.ClassTransformer
  GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 384.76' Renderer: 'GeForce GT 630/PCIe/SSE2'
  Pulsar/inspirations loaded Pulses: - InspirationsShared (Enabled/Forced)
                                     - InspirationsBuilding (Enabled/Not Forced)
                                     - InspirationsUtility (Enabled/Not Forced)
                                     - InspirationsTools (Enabled/Not Forced)
                                     - InspirationsRecipes (Enabled/Not Forced)
                                     - InspirationsTweaks (Enabled/Not Forced)
                                     - InspirationsShared (Enabled/Forced)
                                     - WailaPlugin (Enabled/Not Forced)
  Pulsar/natura loaded Pulses: - NaturaCommons (Enabled/Forced)
                               - NaturaOverworld (Enabled/Not Forced)
                               - NaturaNether (Enabled/Not Forced)
                               - NaturaDecorative (Enabled/Not Forced)
                               - NaturaTools (Enabled/Not Forced)
                               - NaturaEntities (Enabled/Not Forced)
                               - NaturaOredict (Enabled/Forced)
                               - NaturaWorld (Enabled/Not Forced)
  Patchouli open book context: n/a
  Suspected Mods: Unknown
  Launched Version: 1.12.2-forge-14.23.5.2854
  LWJGL: 2.9.4
  OpenGL: GeForce GT 630/PCIe/SSE2 GL version 4.5.0 NVIDIA 384.76, NVIDIA Corporation
  GL Caps: Using GL 1.3 multitexturing.
           Using GL 1.3 texture combiners.
           Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
           Shaders are available because OpenGL 2.1 is supported.
           VBOs are available because OpenGL 1.5 is supported.
  Using VBOs: Yes
  Is Modded: Definitely; Client brand changed to 'fml,forge'
  Type: Client (map_client.txt)
  Resource Packs: 
  Current Language: English (US)
  Profiler Position: N/A (disabled)
  CPU: 4x Intel(R) Core(TM) i3 CPU 550 @ 3.20GHz
  Client Crashes Since Restart: 2
  Integrated Server Crashes Since Restart: 0

```

**Comment by BlueAgent** [comment]
I can't seem to reproduce this on 1.12.2 using Plonk v1.0.0.
Can you try reproduce this with only Plonk installed?
Edit: Will add some code to try fix this, it might cause weird selection box but at least it won't crash.

**Comment by BlueAgent** [comment]
Released the fix here: [Plonk v5.0.0 (mc 1.12.2)](https://www.curseforge.com/minecraft/mc-mods/plonk/files/3086466)
Let me know if it still happens.

**Comment by k1r0s** [comment]
seems to be working just fine, huge thanks!
![image](https://user-images.githubusercontent.com/6052309/96492598-2e5f3680-1244-11eb-9d39-d1115f42cc58.png)
